### PR TITLE
Add/wp scripts babel

### PR DESCRIPTION
**Pull request by nosolosw**
Testbed for https://github.com/WordPress/gutenberg/pull/14168

### Testing

* `cd wp-scripts`
* `npm install` (note that this requires having Gutenberg locally checked to the branch of the PR above).
* `npm run build`
* Verify that the file is built, activate the plugin in your WordPress environment and test that it works (look for a block titled ♥ )

### CODE REVIEW

File: wp-scripts/package.json [hunk 1]
```diff
@@ -18,15 +18,6 @@
...
-    "@babel/core": "^7.3.4",
-    "@babel/plugin-proposal-async-generator-functions": "^7.2.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.3.4",
-    "@babel/plugin-transform-react-jsx": "^7.3.0",
-    "@babel/plugin-transform-runtime": "^7.3.4",
-    "@babel/preset-env": "^7.3.4",
-    "@wordpress/babel-plugin-import-jsx-pragma": "file:../../gutenberg/packages/babel-plugin-import-jsx-pragma",
-    "@wordpress/babel-preset-default": "file:../../gutenberg/packages/babel-preset-default",
-    "@wordpress/browserslist-config": "file:../../gutenberg/packages/browserslist-config",
     "@wordpress/scripts": "file:../../gutenberg/packages/scripts"
```
Comment (gziolo) [inline wp-scripts/package.json]: Isn't it sweet? :)

Comment (nosolosw) [reply wp-scripts/package.json]: It is!

Comment (mkaz) [reply wp-scripts/package.json]: :sparkling_heart:

Comment (iandunn) [reply wp-scripts/package.json]: ❤️

**Pull request merged**